test/src/Provider/GoogleTest.php

```diff
@@ -111,18 +110,17 @@ public function testUserData()
 
         $token = $this->mockAccessToken();
 
-        $provider = Phony::partialMock(GoogleProvider::class);
-        $provider->fetchResourceOwnerDetails->returns($response);
-        $google = $provider->get();
+        $google = $this->getMockBuilder(GoogleProvider::class)
+            ->setMethods(['fetchResourceOwnerDetails'])
+            ->getMock();
```

Comment:
Please revert this. I know that PHPUnit can do mocking, but Phony provides a much nicer syntax.